compiler/AST/expr.cpp

```diff
@@ -5858,124 +5885,112 @@ GenRet CallExpr::codegenPrimMove() {
   return ret;
 }
 
-bool CallExpr::codegenPrimMoveRhsIsSpecialPrimop() {
-  CallExpr* rhs    = toCallExpr(get(2));
+static bool codegenIsSpecialPrimitive(BaseAST* target, Expr* e, GenRet& ret) {
```

Comment:
codegenPrimMoveRhsIsSpecialPrimop had a name that was long enough to describe what the function did, but codegenIsSpecialPrimitive does not. Could you add a comment above this function declaration describing what the function does, what its arguments mean, and in what situations it is called?